src/main/java/bio/terra/tanagra/query/UnionQuery.java

```diff
@@ -0,0 +1,19 @@
+package bio.terra.tanagra.query;
```

Comment:
I'm curious about the choice to pull this into its own "tanagra" package. Why not pull this into the datarepo code maybe under a cohort builder folder (Or something similar)

Reply:
I agree, the tanagra name isn't necessary here, I can pick a name better matched to its usage.